dts/arm/st/f1/stm32f100XX.dtsi

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2017 I-SENSE group of ICCS
+ * Copyright (c) 2020 Jonas Eriksson, Up to Code AB
+ *
+ * SoC device tree include for STM32F100xx SoCs
+ * where 'xx' is replaced with the specific SoC model, {C,R,V}{4,6,8,B}
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/f1/stm32f1.dtsi>
+
+/ {
+	sram0: memory@20000000 {
+		reg = <0x20000000 DT_SIZE_K(8)>;
+	};
+
+	soc {
+		flash-controller@40022000 {
+			flash0: flash@8000000 {
+				reg = <0x08000000 DT_SIZE_K(128)>;
+				erase-block-size = <DT_SIZE_K(1)>;
+			};
+		};
+
+		/* spi2 is present only on stm32f100x8 and -xb, remove for -x4 and -x6.
```

Comment:
By restricting file to Xb, you don't need to specify this anymore

Reply:
Correct, fixed